spec/lib/evss/disability_compensation_form/service_spec.rb

```diff
@@ -58,6 +58,22 @@
       end
     end
 
+    context 'with valid BDD input' do
+      let(:valid_bdd_form_content) do
+        File.read 'spec/support/disability_compensation_form/bdd_fe_submission.json'
+      end
+
+      it 'returns a form submit response object' do
+        # TODO: this cassette seems hand-written and is missing a body
```

Comment:
This cassette is bad, but it's been that way for a while.  I'd like to go back and fix it, but I'd prefer to get this merged first because this big cleanup PR  https://github.com/department-of-veterans-affairs/vets-api/pull/4514 needs this PR to be merged first to prevent a coverage drop;\.

Reply:
Could be hand written. Here's the commit history, hard to tell though: https://github.com/department-of-veterans-affairs/vets-api/commits/master/spec/support/vcr_cassettes/evss/disability_compensation_form/submit_form_v2.yml